lldb/test/API/macosx/queues/TestQueues.py

```diff
@@ -456,8 +457,9 @@ def queues_with_libBacktraceRecording(self):
             "doing_the_work_2",
             "queue 2's pending item #0 should be doing_the_work_2",
         )
-        self.assertTrue(
-            queue_performer_2.GetPendingItemAtIndex(9999).IsValid() == False,
+        self.assertEqual(
+            queue_performer_2.GetPendingItemAtIndex(9999).IsValid(),
```

Comment:
Might be useful to do an additional pass over these later. I think we could use `assertFalse` here, for example.

Reply:
Good point -- I can check for this kind of case as a followup.

Reply:
https://github.com/llvm/llvm-project/pull/82526 includes this and more